router/http/src/test/scala/io/buoyant/router/http/PathIdentifierTest.scala

```diff
@@ -71,7 +71,8 @@ class PathIdentifierTest extends FunSuite with Awaits with Exceptions {
       "/mysvc/subsvc/path1/" -> ExpParsedPath("/http/mysvc/subsvc", "/path1/"),
       "/mysvc/subsvc/path1" -> ExpParsedPath("/http/mysvc/subsvc", "/path1"),
       "/mysvc/subsvc/path1/path2" -> ExpParsedPath("/http/mysvc/subsvc", "/path1/path2"),
-      "/mysvc/subsvc/path1/path2/" -> ExpParsedPath("/http/mysvc/subsvc", "/path1/path2/")
+      "/mysvc/subsvc/path1/path2/" -> ExpParsedPath("/http/mysvc/subsvc", "/path1/path2/"),
+      "/mysvc/subsvc/path1/?foo=bar" -> ExpParsedPath("/http/mysvc/subsvc", "/path1/?foo=bar")
```

Comment:
Would you mind adding a test like `/mysvc/subsvc/path1?foo=bar/ -> /path1?foo=bar/`?